app/Traits/UpdateUserSpacesFromSUMS.php

```diff
@@ -2,6 +2,8 @@
 
 declare(strict_types=1);
 
+// phpcs:disable Generic.NamingConventions.TraitNameSuffix.Missing
```

Comment:
I'd probably ignore these in the top-level config.

Reply:
Agreed.